packages/E2ETest/windows/ReactUWPTestApp/Assets/TreeDump/ControlStyleRoundBorder.txt

```diff
@@ -4,848 +4,921 @@
+                            BorderThickness=0,0,0,0
+                            Clip=[NULL]
+                            CornerRadius=0,0,0,0
+                            FlowDirection=LeftToRight
+                            HorizontalAlignment=Stretch
+                            Margin=0,0,0,0
+                            Padding=0,0,0,0
+                            RenderSize=780,30
+                            VerticalAlignment=Stretch
+                            Visibility=Visible
+                        [Windows.UI.Xaml.Controls.Primitives.ScrollBar]
+                            Background=#00FFFFFF
+                            BorderBrush=#00FFFFFF
+                            BorderThickness=0,0,0,0
+                            Clip=[NULL]
+                            CornerRadius=2,2,2,2
+                            FlowDirection=LeftToRight
+                            Foreground=#00FFFFFF
+                            HorizontalAlignment=Right
+                            Margin=0,0,0,0
+                            Padding=0,0,0,0
+                            RenderSize=0,0
+                            VerticalAlignment=Stretch
+                            Visibility=Collapsed
+                        [Windows.UI.Xaml.Controls.Primitives.ScrollBar]
+                            Background=#00FFFFFF
+                            BorderBrush=#00FFFFFF
+                            BorderThickness=0,0,0,0
+                            Clip=[NULL]
+                            CornerRadius=2,2,2,2
+                            FlowDirection=LeftToRight
+                            Foreground=#00FFFFFF
+                            HorizontalAlignment=Stretch
+                            Margin=0,0,0,0
+                            Padding=0,0,0,0
+                            RenderSize=0,0
+                            VerticalAlignment=Stretch
+                            Visibility=Collapsed
+                        [Windows.UI.Xaml.Controls.Border]
+                            Background=#FFE6E6E6
+                            BorderBrush=[NULL]
+                            BorderThickness=0,0,0,0
+                            Clip=[NULL]
+                            CornerRadius=0,0,0,0
+                            FlowDirection=LeftToRight
+                            HorizontalAlignment=Stretch
+                            Margin=0,0,0,0
+                            Padding=0,0,0,0
+                            RenderSize=0,0
+                            VerticalAlignment=Stretch
+                            Visibility=Collapsed
+            [Windows.UI.Xaml.Controls.TextBlock]
                 Clip=[NULL]
-                CornerRadius=2,2,2,2
-                Foreground=#00FFFFFF
-                HorizontalAlignment=Right
-                Margin=0,0,0,0
-                Padding=0,0,0,0
-                RenderSize=0,0
+                FlowDirection=LeftToRight
+                Foreground=#B3E1E1E1
```

Comment:
>Foreground=#B3E1E1E1 [](start = 16, length = 20)

Hmmm I'm not sure why this color changed.

Reply:
so I'm finding that some of these colors are accent colors. Some of these sizes are changed too and I'm trying to find out why.

One thing that we should note is that the lab machine runs 17763 which is RS5. It's possible changes we depend on might be only available in 19h1+?